cookbooks/aws-parallelcluster-slurm/test/controls/config_health_check_spec.rb

```diff
@@ -52,7 +52,7 @@
 
   if instance.graphic?
     if instance.dcgmi_gpu_accel_supported?
-      if os_properties.arm? && (os_properties.centos7? || os_properties.alinux2?)
+      if (os_properties.arm? && (os_properties.centos7? || os_properties.alinux2?)) || (os_properties.alinux2? && node['cluster']['nvidia']['enabled'] == 'no')
```

Comment:
We should define `nvidia_enabled?` in the `cookbooks/aws-parallelcluster-shared/test/libraries/instance.rb` to avoid checking node properties here and there. 

In this way if we change the logic of enabling nvidia we only need to change the nvidia_enabled? function. 

For the code we already have it in `cookbooks/aws-parallelcluster-platform/libraries/nvidia.rb`.